cms/djangoapps/contentstore/views/tests/test_programs.py

```diff
@@ -21,6 +21,8 @@ def setUp(self):
 
         ClientFactory(name=ProgramsApiConfig.OAUTH2_CLIENT_NAME, client_type=CONFIDENTIAL)
 
+        self.create_programs_config()
```

Comment:
This is done in each test. Why is it also in the setUp? I think doing this explicitly in each test is cleaner, since it makes the configuration being used for the clear.